Makefile

```diff
@@ -17,15 +17,14 @@
 # limitations under the License.
 
 extensions = \
-	py3.7 \
 	bioc \
 	r \
 	cuda-9.2 \
 	cuda-10.0-tf-1.14
 
 DOCKER_PREFIX?=renku/renkulab
 DOCKER_LABEL?=latest
-JUPYTERHUB_VERSION?=1.2
+JUPYTERHUB_VERSION?=1.1.0
```

Comment:
the jupyterhub version is pinned in the new base, right? https://hub.docker.com/r/jupyter/base-notebook/dockerfile